### DISCUSSION TITLE
electron issuse

### DISCUSSION THREAD

**Discussion by NightMingGit** [TypeFox/monaco-languageclient#474] [General]
if i use in electron  these code is must?
```
`StandaloneServices.initialize({
    ...getNotificationServiceOverride(document.body),
    ...getDialogServiceOverride()
});

buildWorkerDefinition('../../../node_modules/monaco-editor-workers/dist/workers/', new URL('', window.location.href).href, false);
```

**Comment by kaisalmen** [comment]
I moved this to the "Discussions" as this is not an issue.

`StandaloneServices.initialize` allows you to override monaco services with the ones available from `monaco-vscode-api`. This is not required, but optional.

`buildWorkerDefinition` is just a helper function that allows to load monaco-editor workers more easily. I don't know if it will work as such in an electron context.
